db/db_impl.cc

```diff
@@ -3060,13 +3058,15 @@ Status DBImpl::VerifyChecksum() {
   for (auto& sv : sv_list) {
     VersionStorageInfo* vstorage = sv->current->storage_info();
     ColumnFamilyData* cfd = sv->current->cfd();
+    Options opts = Options(BuildDBOptions(immutable_db_options_,
```

Comment:
`mutable_db_options_` need to access while holding db mutex. You can call DB::GetDBOptions() instead.

Reply:
updated